### ISSUE TITLE
Use of undeclared type 'CardHightlight' when implementing the Getting Started example

### ISSUE THREAD

**Issue by wqfan** [PaoloCuscela/Cards#10]
Hello, thanks for the awesome framework!

I just tried the Getting Started example. After dragging the Card into the ViewController and adding an @IBOutlet. I got an error that "Use of undeclared type 'CardHightlight'".

I installed the framework using Cocoapods. I'm new to iOS dev and I probably miss some stuff. Could you explain a bit why does this happen?

Thanks!

**Comment by PaoloCuscela** [comment]
Thank you very much :D
Try to check the attribute inspector and make sure the "module" field is set to Cards

**Comment by wqfan** [comment]
Thanks for the quick response!

It's set to "Cards", but I still got this error.

<img width="1440" alt="screen shot 2017-10-27 at 7 37 56 am" src="https://user-images.githubusercontent.com/7760864/32102399-1f266256-baea-11e7-98fc-802a6f2af82a.png">

**Comment by PaoloCuscela** [comment]
Got it. 
You have to do "import Cards" in the top of the file. I'll add this to the readme.
Thank you :)

**Comment by wqfan** [comment]
I also tried that. But there is an error "Cannot load underlying module for 'Cards'".

**Comment by wqfan** [comment]
I cleaned the project and then rebuild it. "Cannot load underlying module for 'Cards'" disappears, but I still have the "Use of undeclared type 'CardHightlight'" error.

<img width="1440" alt="screen shot 2017-10-27 at 7 58 38 am" src="https://user-images.githubusercontent.com/7760864/32103056-e70d62d6-baec-11e7-8417-62a26e86aab2.png">

**Comment by wqfan** [comment]
Hello @PaoloCuscela, sorry for the random messages. 

I just found I installed Cards 1.1 instead of the newest 1.2.3. After upgrading to the 1.2.3, the undeclared problem disappears but I got this error: 

IB Designables: Failed to render and update auto layout status for ViewController (BYZ-38-t0r): dlopen(Cards.framework, 1): Library not loaded: @rpath/Player.framework/Player
  Referenced from: Cards.framework
  Reason: image not found

<img width="1440" alt="screen shot 2017-10-27 at 8 14 09 am" src="https://user-images.githubusercontent.com/7760864/32103660-6d532e5a-baef-11e7-88fc-cdcd9058a19c.png">

**Comment by PaoloCuscela** [comment]
Looks strange. I've done a test right now and all seems to work well.
Try this:
New xcode project
Pod init
Add "pod 'Cards'"
Pod install
Pod update
Open 'project.workspace'
Add UIView in storyboard 
Class 'CardHighlight'
Module 'Cards'

**Comment by wqfan** [comment]
Still got the same IB Designables issue. I'm working on Xcode Version 9.0.1 (9A1004). Probably an issue with my Xcode?

<img width="1512" alt="screen shot 2017-10-27 at 10 00 03 am" src="https://user-images.githubusercontent.com/7760864/32108188-e37c8cc0-bafe-11e7-8677-7ff9bb34f234.png">

**Comment by PaoloCuscela** [comment]
Can you post your sample project so I can have a look ?

**Comment by wqfan** [comment]
Sure! Thank you very much for your help!
[project.zip](https://github.com/PaoloCuscela/Cards/files/1422450/project.zip)

**Comment by PaoloCuscela** [comment]
![schermata 2017-10-27 alle 17 18 06](https://user-images.githubusercontent.com/23332257/32111502-f72fcc96-bb3a-11e7-9992-173bc06fef3e.png)

It's  working fine here. I'm on 9.0.1 too ... 🤔
Maybe reinstalling XCode ?

**Comment by wqfan** [comment]
Thank you @PaoloCuscela. Sorry I forget to tell you that I'm able to build the project now, though it has a storyboard error. Before, when I'm using Cards 1.1, the prototype of a card can appear on the storyboard, but currently it can't show up, so I think it's that error causes the UI to fail to display.

**Comment by PaoloCuscela** [comment]
I googled about it yesterday. It turned out that is a common issue, probably about some optional vars stopping IBDesignables to be rendered.
I'll try to fix that in these days

**Comment by wqfan** [comment]
Thanks for your efforts!

**Comment by PaoloCuscela** [comment]
Good news :D 
v1.2.4 out. 
Now it should display the preview on the storyboard correctly.

**Comment by wqfan** [comment]
Sorry @PaoloCuscela I missed your last comment. I just upgraded it to 1.2.4 and the preview is still blank. Probably it's a weird problem happened on my macOS only. Thanks very much for your help!

**Comment by PaoloCuscela** [comment]
Already tried with Editor -> Refresh all views ?

**Comment by wqfan** [comment]
Yes. And the storyboard error is still here:
```
Designables: Failed to render and update auto layout status for ViewController (BYZ-38-t0r): dlopen(Cards.framework, 1): Library not loaded: @rpath/Player.framework/Player
  Referenced from: Cards.framework
  Reason: image not found
```

**Comment by PaoloCuscela** [comment]
Do you get this error only with my repo ?

**Comment by wqfan** [comment]
Yes. And I find it works fine on your Demo project. I'm very confused why the error happens on my projects.

**Comment by wqfan** [comment]
Hello @PaoloCuscela, I upgraded the Xcode to the newest 9.1. The storyboard error disappeared but I still can't see the preview.

<img width="1424" alt="screen shot 2017-10-31 at 3 25 03 pm" src="https://user-images.githubusercontent.com/7760864/32244711-d93a9c80-be4f-11e7-9f1b-2eff3a68783d.png">

**Comment by FranciscoPereira** [comment]
I'm having the same error too but in my case I'm trying to use it on a UICollectionViewCell by adding a subview with Card class type.

**Comment by PaoloCuscela** [comment]
I've done some testing.
Looks really strange, with the sdk installed from cocoapods repo preview are not rendered.
With a local copy ( ``pod 'Cards', :path => '../' ``) it's all fine.
I've to do some research on it.

**Comment by AshleyDeng** [comment]
We have the same issue as @wqfan reported. There is an error on Storyboard saying

`IB Designables: Failed to render and update auto layout status for EventsTableViewController (ZmE-MU-FrL): dlopen(Cards.framework, 1): Library not loaded: @rpath/Player.framework/Player
  Referenced from: Cards.framework
  Reason: image not found`

The UI was not able to be rendered on Storyboard neither.

Me and my guy are on `Cards 1.3.0` and `XCode 9.1 and 9.2`

**Comment by PaoloCuscela** [comment]
I found a workaround that might work, try adding this in your podfile.
https://github.com/CocoaPods/CocoaPods/issues/5334#issuecomment-254199946

**Comment by avjiang** [comment]
Hi, just found out this library and i face the same issue as stated above, failed to render IB designable error. is there any update for this issue?

Thanks!

**Comment by MhndMousa** [comment]
I had the same error trying to use cocoapods, I worked around it by adding the library manually.

**Comment by dannylnz** [comment]
Hi, any update on this bug? 
does anyone found a workaround for this? 
Thanks 👍

**Comment by lukaskirner** [comment]
I still get this bug in 1.3.2. With pod I get the IBDesignables error and if I install it manually it wont find the Cards module.
Is there no fix out there for any of these problems?

**Comment by QinGuan2004** [comment]
This issue is not because of the Cards its due to the latest Cocoapods version to fix it just downgrade your cp to about 1.4< and it shouldnt show the error anymore 

@lukaskirner if you install it manually from what i know you wouldnt need to import Cards like its already imported somehow

**Comment by lukaskirner** [comment]
@QinGuan2004 suggested to downgrade Cocoapods which didn't completly removed the problem but after I added this little code to the Podfile it workes.

```
post_install do |installer|
  installer.pods_project.build_configurations.each do |config|
    config.build_settings['LD_RUNPATH_SEARCH_PATHS'] = [
      '$(FRAMEWORK_SEARCH_PATHS)'
    ]
  end
end
```

**Comment by michaelcarroll** [comment]
I had this same issue and fixed it by adding a signature fix and the code @lukaskirner mentioned above.

Here is what my pod file looks like:

```
# Uncomment the next line to define a global platform for your project
# platform :ios, '11.0'

target 'CardTest' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for CardTest
	pod 'Cards'

post_install do |installer|
  installer.pods_project.build_configurations.each do |config|
    config.build_settings['LD_RUNPATH_SEARCH_PATHS'] = [
      '$(FRAMEWORK_SEARCH_PATHS)'
    ]
    installer.pods_project.build_configurations.each do |config|
        config.build_settings.delete('CODE_SIGNING_ALLOWED')
        config.build_settings.delete('CODE_SIGNING_REQUIRED')
   end
end
end
end
```

**Comment by jctaoo** [comment]
It works for me.
xcode: 11.3
macOS: 10.15
CocoaPods: 1.8.1

> @QinGuan2004 suggested to downgrade Cocoapods which didn't completly removed the problem but after I added this little code to the Podfile it workes.
> 
> ```
> post_install do |installer|
>   installer.pods_project.build_configurations.each do |config|
>     config.build_settings['LD_RUNPATH_SEARCH_PATHS'] = [
>       '$(FRAMEWORK_SEARCH_PATHS)'
>     ]
>   end
> end
> ```
